### PR TITLE
HOTFIX: Hide review metadata for non-admin non-copilot users when getting submission details

### PR DESCRIPTION
**Pull request by meshde**

### CODE REVIEW

**Comment by callmekatootie** [comment]
@cwdcwd The hotfix is ready to be merged

File: src/common/helper.js [hunk 1]
```diff
@@ -554,6 +554,33 @@ function * postToBusApi (payload) {
...
+ * Function to remove metadata details from reviews for members who shouldn't see them
+ * @param  {Array} reviews The reviews to remove metadata from
+ * @param  {Object} authUser The authenticated user details
+ */
+function cleanseReviews (reviews, authUser) {
+  const cleansedReviews = []
+
+  if (!authUser.scopes) {
+    _.forEach(reviews, (review) => {
+      const admin = _.filter(authUser.roles, role => role.toLowerCase() === 'Administrator'.toLowerCase())
```
Comment (dushyantb) [inline src/common/helper.js]: why we are checking the role for each review? the role would remain the same for the entire process.
hence, we should first check the role and then filter out the review

Comment (callmekatootie) [reply src/common/helper.js]: Makes sense.

**Comment by callmekatootie** [comment]
@dushyantb The issue reported is now resolved

**Comment by dushyantb** [comment]
looks ok... merging now

**Pull request merged**